### DISCUSSION TITLE
German Railways

### DISCUSSION THREAD

**Discussion by jschwartzenberg** [dosemu2/dosemu2#2142] [General]
Maybe somebody could introduce them to DOSEMU2 so they can move to newer hardware? :)
https://www.tomshardware.com/software/windows/ms-dos-and-windows-311-still-run-train-dashboards-at-german-railway-company-listed-admin-job-for-30-year-old-operating-system

**Comment by stsp** [comment]
:)
So introduce them to something that
will force them to upgrade the hardware?
Sounds quite microsoftish. :)
Likely they don't want to upgrade their
hardware. And dosemu2 will likely not
run on a "systems with 166MHz processors
and a whopping 8MB of RAM".

**Reply by jschwartzenberg** [reply]
I think broken hardware may actually force the upgrade at on point or another and dosemu2 would actually enable them to do it ;)

**Comment by jharrison03** [comment]
I saw this too only on a different site.  I assume whatever they're doing the position isn't about software that is directly running the trains.

**Comment by stsp** [comment]
So I guess the only way to introduce
them to dosemu2, is for someone from
here to hire to that position. :)
